### PR TITLE
Theme Cicada

### PR DESCRIPTION
**Pull request by marjohn56**
Initial Commit.

Checked this over and it appears to be good. There are sufficient differences between this theme and Rebellion to have both. This theme uses different fonts, some colours are different etc.

### CODE REVIEW

File: misc/theme-cicada/Makefile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+PLUGIN_NAME=		theme-cicada
+PLUGIN_VERSION=		0.2
+PLUGIN_REVISION=	2
+PLUGIN_COMMENT=		The cicada theme - grey/orange
+PLUGIN_MAINTAINER=	remic-webdesign@chello.at
```
Comment (fabianfrz) [inline misc/theme-cicada/Makefile]: @marjohn56 If you push it, aren't you the maintainer?

File: misc/theme-cicada/src/opnsense/www/themes/cicada/assets/stylesheets/bootstrap-dialog.less [hunk 2]
```diff
@@ -0,0 +1,132 @@
+.modal-backdrop {
+    z-index: -1;
+}
+.bootstrap-dialog {
+    /*** Icon animation* Copied from font-awesome: http://fontawesome.io;
```
Comment (fabianfrz) [inline misc/theme-cicada/src/opnsense/www/themes/cicada/assets/stylesheets/bootstrap-dialog.less]: this comment seems to be never closed

File: misc/theme-cicada/src/opnsense/www/themes/cicada/assets/stylesheets/bootstrap-select/less/bootstrap-select.less [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+    Bootstrap-select {
+        v1.9.3 {
+            (http://silviomoreto.github.io/bootstrap-select)** {
+                Copyright {
+                    2013-2015 {
+                        bootstrap-select* {
+                            Licensed {
+                                under {
+                                    MIT {
+                                        (https://github.com/silviomoreto/bootstrap-select/blob/master/LICENSE)*/span.form-control {
```
Comment (fabianfrz) [inline misc/theme-cicada/src/opnsense/www/themes/cicada/assets/stylesheets/bootstrap-select/less/bootstrap-select.less]: format

File: misc/theme-cicada/Makefile (hunk 1 above)
Comment (marjohn56) [reply misc/theme-cicada/Makefile]: Very good question, to which I do not know the simple answer. ;)

I'm just the pipe... I'll change this to me and credit remic-*** in the comment.

File: misc/theme-cicada/src/opnsense/www/themes/cicada/assets/stylesheets/bootstrap-dialog.less (hunk 2 above)
Comment (marjohn56) [reply misc/theme-cicada/src/opnsense/www/themes/cicada/assets/stylesheets/bootstrap-dialog.less]: Does not seem to affect anything oddly enough, But I'll correct it.

File: misc/theme-cicada/src/opnsense/www/themes/cicada/assets/stylesheets/bootstrap-select/less/bootstrap-select.less (hunk 3 above)
Comment (marjohn56) [reply misc/theme-cicada/src/opnsense/www/themes/cicada/assets/stylesheets/bootstrap-select/less/bootstrap-select.less]: Thanks @fabianfrz  - I'll correct these this morning.

File: misc/theme-cicada/Makefile [hunk 4]
```diff
@@ -0,0 +1,8 @@
+PLUGIN_NAME=		theme-cicada
+PLUGIN_VERSION=		0.2
```
Comment (fichtner) [inline misc/theme-cicada/Makefile]: Let's start at 0.1 and clear the PLUGIN_REVISION :)

File: misc/theme-cicada/Makefile (hunk 1 above)
Comment (fichtner) [reply misc/theme-cicada/Makefile]: Yes please, we can pass it over later during a 1.x transition when he handles the PRs.

File: misc/theme-cicada/Makefile (hunk 4 above)
Comment (marjohn56) [reply misc/theme-cicada/Makefile]: Is done..

Note sure how to to tidy of the format in bootstrap-select.less. Whichever way I do it looks naff.

**Comment by marjohn56** [comment]
Done apart from the format issue for bootstrap-select.less. No matter which way I try and tidy that it still looks wrong.

**Comment by marjohn56** [comment]
Updated comments.

File: misc/theme-cicada/src/opnsense/www/themes/cicada/assets/stylesheets/bootstrap-select/less/bootstrap-select.less (hunk 3 above)
Comment (marjohn56) [reply misc/theme-cicada/src/opnsense/www/themes/cicada/assets/stylesheets/bootstrap-select/less/bootstrap-select.less]: No idea... as I said bellow.

Done apart from the format issue for bootstrap-select.less. No matter which way I try and tidy that it still looks wrong.

Comment (fichtner) [reply misc/theme-cicada/src/opnsense/www/themes/cicada/assets/stylesheets/bootstrap-select/less/bootstrap-select.less]: let me take a look post merge, thanks!

**Pull request merged**